README.md

```diff
@@ -3,4 +3,4 @@
 [![license](https://img.shields.io/github/license/cloudigrade/cloudigrade.svg)]()
 [![Build Status](https://travis-ci.org/cloudigrade/cloudigrade.svg?branch=master)](https://travis-ci.org/cloudigrade/cloudigrade)
 [![Coverage Status](https://coveralls.io/repos/github/cloudigrade/cloudigrade/badge.svg?branch=coverage-configs)](https://coveralls.io/github/cloudigrade/cloudigrade?branch=coverage-configs)
-[![Scrum Board](https://raw.githubusercontent.com/ZenHubIO/support/master/zenhub-badge.svg)](https://app.zenhub.com/workspace/o/cloudigrade/cloudigrade/boards?repos=118967787)
+[![Scrum Board](https://img.shields.io/badge/Shipping_faster_with-ZenHub-5e60ba.svg?style=flat)](https://app.zenhub.com/workspace/o/cloudigrade/cloudigrade/boards?repos=118967787)
```

Comment:
Can we have a spicier tagline than this?

Reply:
If we're going to link to them, I figured we should keep their official tagline that's in their badge.